test/test-agent-stopped.js

```diff
@@ -43,7 +43,7 @@ describe('test-agent-stopped', function() {
       assert.ok(!agent.isActive());
       scope.done();
       done();
-    }, 20);
+    }, 200);
```

Comment:
Nit: There's a comment above this that says `Wait 20ms` and it should say `200ms` instead.